identity-service/src/routes/trackListens.js

```diff
@@ -214,11 +216,12 @@ module.exports = function (app) {
     // Those listened should NOT be recorded in the userTrackListen table
     if (!isNaN(req.body.userId)) {
       // Find / Create the record of the user listening to the track
-      const [userTrackListenRecord, created] = await models.UserTrackListen
+      const [userTrackListenRecord, created] = await models.UserTrackListens
         .findOrCreate({ where: { userId: req.body.userId, trackId } })
 
       // If the recrod was not created, updated the timestamp
       if (!created) {
+        userTrackListenRecord.increment('count')
```

Comment:
Add `await` - This function returns a promise.
If this happens, I think you can remove the next couple of lines to update the 'updatedAt' field as incrementing should do it (I believe).

Reply:
thx :)